src/components/RegistrationForm.spec.tsx

```diff
@@ -1,13 +1,13 @@
 import { mount } from '@cypress/react';
-import { Provider, defaultTheme } from '@adobe/react-spectrum';
+import { defaultTheme, Provider } from '@adobe/react-spectrum';
 
 import RegistrationForm from './RegistrationForm';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const mountWithTheme = (elem) : any => mount(
     <Provider theme={ defaultTheme }>
         { elem }
-    </Provider>
+    </Provider>,
```

Comment:
I'm not a fan of dangling commas in this special case, where the only argument to a function is a JSX element. But it seems like it's a very rare case, so let's leave it like this and see how it feels in the long run. 😊